main.tf

```diff
@@ -202,10 +203,10 @@ module "kubeconfig" {
   depends_on = [module.eks.cluster_id] # The name/id of the EKS cluster. Will block on cluster creation until the cluster is really ready.
 }
 
-# Database Setup - https://registry.terraform.io/modules/terraform-aws-modules/rds/aws/3.3.0
+# Database Setup - https://registry.terraform.io/modules/terraform-aws-modules/rds/aws/5.9.0
 module "postgresql" {
   source  = "terraform-aws-modules/rds/aws"
-  version = "3.3.0"
+  version = "5.9.0"
```

Comment:
Major version updates here. Make sure there are no breaking changes on our side.

Reply:
I will re-check this module update for breaking changes, no issues have been encountered so far with externally created postres instances during testing.